pkg/tsdb/elasticsearch/client/search_request.go

```diff
@@ -9,6 +9,7 @@ const (
 	HighlightPreTagsString  = "@HIGHLIGHT@"
 	HighlightPostTagsString = "@/HIGHLIGHT@"
 	HighlightFragmentSize   = 2147483647
+	DefaultPrecision        = 3
```

Comment:
Just to be slightly more precisive, what do you think about adding "geoHash" to the name?

```suggestion
	DefaultGeoHashPrecision        = 3
```

Reply:
Fixed in https://github.com/grafana/grafana/pull/73922/commits/fffe7471ead9d952a14796d6558680e591c4b989